docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '2.1'
+volumes:
+  resin-data: {}
+services:
+  balena-supervisor:
+    build: ./
+    privileged: true
+    tty: true
+    restart: always
+    network_mode: host
+    environment:
+      - DOCKER_ROOT=/mnt/root/var/lib/docker
+      - DOCKER_SOCKET=/var/run/balena-engine.sock
+#       - "BOOT_MOUNTPOINT=$BOOT_MOUNTPOINT"                       <--- Must be set as service variable
+#       - "DELTA_ENDPOINT=$DELTA_ENDPOINT"                         <--- Must be set as a service variable
+#       - "LED_FILE=${LED_FILE}"                                   <--- Must be set as a service variable
+#       - "LISTEN_PORT=$LISTEN_PORT"                               <--- Must be set as a service variable
+#       - "NODE_EXTRA_CA_CERTS=${NODE_EXTRA_CA_CERTS}"             <--- Must be set as a service variable
+#       - "SUPERVISOR_IMAGE=${SUPERVISOR_IMAGE}:${SUPERVISOR_TAG}" <--- Must be set as aservice variable
+    volumes:
+      - 'resin-data:/data'
+#       - type: bind                                               <--- Bind mounts are not allowed
```

Comment:
Not sure if it's useful to add the bind mounts as comments

Reply:
I added them since those are needed to run the Supervisor and I've already gone through the work of adding what we need to actually run the Supervisor with this docker-compose file we just have to do whatever is required now to allow these requirements. Without them we'd have to figure out again what we need.

Reply:
We could remove them if you wanted to. I thought they were good documentation for the future with no harm.